### PR TITLE
SPCTR-2968: Fix - Content Timeline connector icon is not getting updated with the one which was set above

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

When adding new content timeline child, icon is not getting updated with the one which was set above in the editor.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 == Changelog ==
 
+= x.x.x =
+* Fix: Content Timeline - Fixed default icon issue When adding new content timeline child, icon is not getting updated with the one which was set above in the editor.
```
Comment (aapatre) [inline readme.txt:170]: ( 👁️ _ 👁️) 👇  
```suggestion
* Fix: Content Timeline - Fixed default icon issue when adding new content timeline child, icon is not getting updated with the one which was set above in the editor.
```

Comment (rajmohanreddy774) [reply readme.txt]: updated

File: README.md [hunk 2]
```diff
@@ -166,10 +166,12 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Form - Resolved an issue in which the submit button would appear differently on the frontend when no preset from the variation picker was selected.
+
 ### 2.6.8 - WEDNESDAY, 21st JUNE 2023 ###
 * Fix: Buttons - Resolved an issue in which the background color would appear transparent after installing the WooCommerce plugin.
 * Fix: Call To Action - Resolved an issue wherein an extra margin would appear above the heading of the Call To Action block in the Spectra One theme.
-* Fix: Form - Resolved an issue in which the submit button would appear differently on the frontend when no preset from the variation picker was selected.
```
Comment (AbhijitNage123) [inline README.md:172]: @rajmohanreddy774 Please get change-log entry added back for 2.6.8

File: readme.txt [hunk 3]
```diff
@@ -166,10 +166,12 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 == Changelog ==
 
+= x.x.x =
+* Fix: Form - Resolved an issue in which the submit button would appear differently on the frontend when no preset from the variation picker was selected.
+
 = 2.6.8 - WEDNESDAY, 21st JUNE 2023 =
 * Fix: Buttons - Resolved an issue in which the background color would appear transparent after installing the WooCommerce plugin.
 * Fix: Call To Action - Resolved an issue wherein an extra margin would appear above the heading of the Call To Action block in the Spectra One theme.
-* Fix: Form - Resolved an issue in which the submit button would appear differently on the frontend when no preset from the variation picker was selected.
```
Comment (AbhijitNage123) [inline readme.txt:172]: same here

Comment (rajmohanreddy774) [reply readme.txt]: Changes done.

**Review by AbhijitNage123 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -169,6 +169,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 ### x.x.x ###
 * Fix: Forms - Resolved an issue in which the overall alignment would not work as intended when three forms are placed with different alignments.
 * Fix: Testimonial - Resolved an inssue in which the arrows would overlap the content on mobile devices.
+* Fix: Content Timeline - Fixed default icon issue when adding new content timeline child, icon is not getting updated with the one which was set above in the editor.
```
Comment (bsf-zanev) [inline README.md:172]:
```suggestion
* Fix: Content Timeline - Resolved an issue in which the user-defined icon would not appear on new timeline items in the editor.
```

**Review by bsf-zanev — APPROVED** [review]

**Review by bsf-zanev — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**